source/Nuke.Common/ChangeLog/ReleaseNotes.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Maintainers of NUKE.
+// Distributed under the MIT License.
+// https://github.com/nuke-build/nuke/blob/master/LICENSE
+
+namespace Nuke.Common.ChangeLog
+{
+    using System;
+    using System.Collections.Generic;
+    using JetBrains.Annotations;
+    using NuGet.Versioning;
+
+    [PublicAPI]
+    public class ReleaseNotes
+    {
+        public bool IsEmpty => Notes.Count == 0;
+        public bool Unreleased => Version == null;
+        [CanBeNull] public NuGetVersion Version { get; }
+        public IReadOnlyList<string> Notes { get; }
+        public (int startIdx, int stopIdx) Section { get; }
```

Comment:
Use 2 properties instead of a Value Tuple, also in ctor and where the property is accessed.